pkgs/development/python-modules/sqlfmt/default.nix

```
@@ -0,0 +1,66 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  fetchFromGitHub,
+  importlib-metadata,
+  black,
+  poetry-core,
+  click,
+  jinja2,
+  platformdirs,
+  tomli,
+  tqdm,
+  gitpython,
+}:
+
+buildPythonPackage rec {
+  pname = "sqlfmt";
+  version = "0.23.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "tconbeer";
+    repo = "sqlfmt";
+    rev = "v${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```
This prevents branch/tag mismatch.